config/config.toml.example

```diff
@@ -66,6 +66,11 @@ delay-clean-table-lock = 0
 # Maximum number of the splitting region, which is used by the split region statement.
 split-region-max-num = 1000
 
+# enable-pk-is-handle is used to control alter primary key feature. Default is true, indicate the alter primary key feature is disabled.
```

Comment:
It looks really confused here. Why true means `disable` since the name is `enableXXX`. And the name makes me wondering enable pk to do what.

Reply:
Do you have any advice?

Reply:
How about `change-primary-key`?
There may be a potential problem when each TiDB configuration is not the same. But considering that this may be a temporary solution, it should be fine.

Reply:
Mainly, if this configuration allows, but the table's pkIsHandle is true, we also do not support drop the primary key. I think it still a little different.  @jackysp @lamxTyler

Reply:
Mainly, if this configuration allows, but the table's pkIsHandle is true, we also do not support drop the primary key. I think it still a little different.  @jackysp @lamxTyler

Reply:
I think this difference can be explained in comments and documentation. `pk-is-handle` is not easy for users to understand the meaning.

Reply:
OK. I change it to `enable-alter-primary-key`. @jackysp @lamxTyler

Reply:
After discussing with PM, I change it to `alter-primary-key`.